test/ClangImporter/cxx_interop_sil.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend -module-name cxx_sil -I %S/Inputs/custom-modules -module-cache-path %t -enable-cxx-interop -emit-sil -o - -primary-file %s | %FileCheck %s
+
+import CXXInterop
+
+// CHECK-LABEL: sil hidden @$s7cxx_sil13virtualMethod1as5Int32VSpySo14MethodsVirtualVG_tF
+// CHECK: [[INT_LIT:%.*]] = integer_literal $Builtin.Int32, 7
+// CHECK: [[INT:%.*]] = struct $Int32 ([[INT_LIT]] : $Builtin.Int32)
+// CHECK: [[THIS_PTR:%.*]] = begin_access [modify] [unsafe] {{%.*}} : $*MethodsVirtual
+// CHECK: ([[VIRTUAL_METHOD:%.*]], [[THIS_PTR_ADJUST:%.*]]) = cxx_virtual_method [[THIS_PTR]] : $*MethodsVirtual, #MethodsVirtual.SimpleVirtualMethod!1.foreign : (inout MethodsVirtual) -> (Int32) -> Int32, $@convention(c) (@inout MethodsVirtual, Int32) -> Int32
```

Comment:
Can you add a SIL test that also tests IRGen as well?

Reply:
Just to clarify, is this a new test file that contains SIL and then emits IRGen? This same example code is in both test files (one that filechecks swift and one that filechecks ir). I'm not sure what an additional test would do except test this-ptr-adjustment. Happy to add anyways, but just checking because I was confused.

Reply:
ah ok. I didn't notice that. I have seen people generally put a separate IRGen test in IRGen to make sure people know that there is code coverage of this when looking through the tests. I would be fine if you just copied this and moved the IRGen part of the tests into IRGen.